src/main/resources/messages/en.yml

```diff
@@ -28,12 +28,24 @@
 #                    example: Notch
 # %channelname%:    name of the channel that the message is coming from
 #                    example: server-chat
+# %reply%:          the message displayed when the message is a reply to another message
+#                    message format can be configured with DiscordToMinecraftMessageReplyFormat
+#                    is not included if the message isn't a reply
```

Comment:
Has a grammatical error and is somewhat unclear, I'd suggest something like the below
```suggestion
# %reply%:          the message displayed when the message is a reply to another message.
#                    The message format can be configured with DiscordToMinecraftMessageReplyFormat,
#                    this will be empty if the message isn't a reply to another message
```